training/common/driver-toolkit/Containerfile

```diff
@@ -11,26 +11,25 @@ ARG ENABLE_RT=''
 
 USER root
 
-# TODO: rework this monstrosity into a build.sh (or even not shell script)
 # The need for the `cp /etc/dnf/dnf.conf` is a workaround for https://github.com/containers/bootc/issues/637
-RUN mv /etc/selinux /etc/selinux.tmp \
-    && dnf -y update --exclude kernel* \
-    && dnf info --installed kernel > /dev/null || dnf install -y kernel-core \
-    && if [ "${KERNEL_VERSION}" == "" ]; then \
-        RELEASE=$(dnf info --installed kernel-core | awk -F: '/^Release/{print $2}' | tr -d '[:blank:]') \
-        && VERSION=$(dnf info --installed kernel-core | awk -F: '/^Version/{print $2}' | tr -d '[:blank:]') \
-        && export KERNEL_VERSION="${VERSION}-${RELEASE}" ;\
-        fi \
-    && echo "${KERNEL_VERSION}" \
+RUN if [ "${KERNEL_VERSION}" == "" ] ; then \
```

Comment:
I prefer  
test -z ${KERNEL_VERSION}

Reply:
If this is not a bootable container image ,is there a reason for requiring the kernel?

Reply:
The Driver Toolkit is only used to compile out-of-tree kernel modules. It is used for the `intel-bootc` and `nvidia-bootc` image build with a multi-stage approach. The drivers are built in DriverToolkit, then copied over the final image, which is bootable.